Tools/wasm/emscripten/web_example/index.html

```diff
@@ -4,7 +4,7 @@
         <meta charset="UTF-8" />
         <meta http-equiv="X-UA-Compatible" content="IE=edge" />
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <meta name="author" content="Katie Bell" />
+        <meta name="author" content="Katie Bell, Adam Hartz" />
         <meta name="description" content="Simple REPL for Python WASM" />
```

Comment:
Could you correct the syntax of HTML self-closing tags here? no need for space and slash

See: https://github.com/python/cpython/pull/132220#issue-2976745244